hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -551,14 +551,81 @@ public void deleteSnapshot(String volumeName,
 
   /**
    * List snapshots in a volume/bucket.
-   * @param volumeName volume name
-   * @param bucketName bucket name
-   * @return list of snapshots for volume/bucket snapshotpath.
+   * @param volumeName     volume name
+   * @param bucketName     bucket name
+   * @param snapshotPrefix snapshot prefix to match
```

Comment:
This is helpful to narrow down the list of snapshots if someone has create few thousand snapshots. Another parameter that could be helpful is to return snapshots that were created  between two time timestamps. It doesn't have to be  in this same PR though. You can file a separate Jira for that.

Reply:
> Another parameter that could be helpful is to return snapshots that were created between two time timestamps.

@prashantpogde We could have another overloaded `listSnapshot()` that takes such params in another jira.

Reply:
Yup.